src/core/annotation.js

```diff
@@ -266,25 +266,19 @@ class AnnotationFactory {
             FreeTextAnnotation.createNewAnnotation(
               xref,
               annotation,
-              results,
               dependencies,
               { evaluator, task, baseFontRef }
             )
           );
           break;
         case AnnotationEditorType.INK:
           promises.push(
-            InkAnnotation.createNewAnnotation(
-              xref,
-              annotation,
-              results,
-              dependencies
-            )
+            InkAnnotation.createNewAnnotation(xref, annotation, dependencies)
           );
       }
     }
 
-    await Promise.all(promises);
+    const results = await Promise.all(promises);
 
     return {
       annotations: results,
```

Comment:
Given the changes here, this could possible be simplified even further now?
```suggestion
    return {
      annotations: await Promise.all(promises),
```